src/main/resources/marc4bibframe.yml

```diff
@@ -33,13 +41,6 @@ fieldRules:
                 data:
                 expression: args.contains(data.substring(22, 23))
                 args: a, b, c, d, e, f, g, j
-        - types: CATEGORY_SET
```

Comment:
these changes that were made in scope of MODLD-368 are incorrect (don't know why tests were passing)
in this PR I've placed the configuration in the correct place (line 23)

Reply:
👍 . Thank You for fixing this.

That is strange. I am also not understanding how the tests were passing earlier. Do you know @aleksei-pronichev ?

Reply:
need to recheck test cases for this situation. (suggest to creat little tech debt)